src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -291,16 +286,11 @@ function ReimbursementAccountPage({route, policy, isLoadingPolicy}: Reimbursemen
             return;
         }
 
-        if (isPastSignerStep && !allAgreementsChecked) {
+        if (isPastSignerStep) {
             setNonUSDBankAccountStep(CONST.NON_USD_BANK_ACCOUNT.STEP.AGREEMENTS);
             return;
         }
 
-        if (isPastSignerStep && allAgreementsChecked) {
-            setNonUSDBankAccountStep(CONST.NON_USD_BANK_ACCOUNT.STEP.DOCUSIGN);
-            return;
-        }
-
```

Comment:
@MrMuzyk just curious why we don't save Docusign step? 🤔

Reply:
Cool. Let me double check

Reply:
It's working 🎉 

Could you check if we can fill optimistic bank account data under `Make or track payments` section without reloading?

https://github.com/user-attachments/assets/77bb2b0f-820e-4fc5-85ee-2515b966177c